be/src/common/config.h

```diff
@@ -482,8 +422,8 @@ CONF_mInt64(write_buffer_size, "104857600");
 // NOTICE(cmy): set these default values very large because we don't want to
 // impact the load performance when user upgrading Doris.
 // user should set these configs properly if necessary.
-CONF_Int64(load_process_max_memory_limit_bytes, "107374182400"); // 100GB
-CONF_Int32(load_process_max_memory_limit_percent, "80");         // 80%
+CONF_mInt64(load_process_max_memory_limit_bytes, "107374182400"); // 100GB
+CONF_mInt32(load_process_max_memory_limit_percent, "80");         // 80%
```

Comment:
These 2 configs are only changed in LoadChannelMgr's init function, which will only be called only once when server start, they are immutable.

Reply:
OK.